src/azure-cli-core/azure/cli/core/azclierror.py

```diff
@@ -25,9 +27,11 @@ class AzCLIError(CLIError):
     """ Base class for all the AzureCLI defined error classes.
     DO NOT raise this error class in your codes. """
 
-    def __init__(self, error_msg, recommendation=None):
+    def __init__(self, error_msg, command=None, recommendation=None):
         # error message
         self.error_msg = error_msg
+        self.command = command
+        self.error_help = None
```

Comment:
Perhaps the name `error_suggestion`  is easier to understand than `error_help`?

Reply:
Done